frontend/src/views/NewAccountView.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+<h2>Create New User</h2>
+	<v-form
```

Comment:
Is this spacing really weird on the file or just how github displayed it?

Reply:
Hmm...it does not look like this locally...I will try reformatting and see if that does anything

Reply:
I am not sure if this is how github is displaying it or it is interpreting 2 spaces as 4 or something...

Reply:
Okay, I think I fixed it. I just had to manually space it rather than tab